check_dnsserials

```diff
@@ -147,8 +151,14 @@ if [ -z "$serial" ]; then
 fi
 verbose "Master serial: $serial"
 
+if [ -n $cache ]; then
+        nameserverL="$cache"
+else
+        nameserverL=$($COMMAND_DIG +short NS @$master $domain)
+fi
+
 # iterate the NS-RRs grabbed from master dns-server for the domain
-for server in $($COMMAND_DIG +short NS @$master $domain); do
+for server in $nameserverL; do
```

Comment:
Could you please enlighten me, what the `L` stands for?

Reply:
It stands for List.

I am open for suggestions :-)